inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -669,7 +674,30 @@ class MKLDNNNode {
 
     bool canBePerformedAsScaleShift(const MKLDNNNode *parentNode = nullptr) const;
 
+    bool isDynamicNode() const {
+        return isDynamic;
+    }
+
+    Shape getOriginalInputShapeAtPort(size_t port) const {
```

Comment:
```suggestion
    Shape getInputShapeAtPort(size_t port) const {
```
Short names is better.

Reply:
done